VideoLocker/src/main/java/org/edx/mobile/module/serverapi/parser/GsonParser.java

```diff
@@ -0,0 +1,34 @@
+package org.edx.mobile.module.serverapi.parser;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+
+import java.io.Reader;
+import java.lang.reflect.Type;
+import java.util.List;
+
+/**
+ * Created by rohan on 2/6/15.
+ */
+class GsonParser implements IParser {
```

Comment:
I'm not really against the abstraction, but is there a case where we'd want to swap in a different parser?